### PR TITLE
Uses Geolocation API to get the users location

### PR DESCRIPTION
**Pull request by akanshaaa19**
This PR fixes issue #3 

When the site is loaded for the first time, the user's current location is detected and the weather for that location is shown.
@akshgpt7

### CODE REVIEW

File: example.env [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_APIKEY=
\ No newline at end of file
+REACT_APP_APIKEY=864bc9e4e54e34b8ac0de0edb53fa68b
```
Comment (ttn-ttn) [inline example.env:1]: You shouldn't commit the api key, it's already in the Netlify environment if I'm not mistaken.

File: package-lock.json
Comment (ttn-ttn) [inline package-lock.json:1]: As far as I know, the package-lock.json should not be committed

File: example.env (hunk 1 above)
Comment (ttn-ttn) [inline example.env:1]: You shouldn't commit the api key, it's already in the Netlify environment if I'm not mistaken.

File: src/App.js [hunk 2]
```diff
@@ -1,20 +1,46 @@
...
+        const url='http://api.openweathermap.org/geo/1.0/reverse?lat=' + position.coords.latitude + '&lon=' + position.coords.longitude + '&limit=5&appid=' + apiKey
+        
+        fetch(
+          url).then((result) => result.json())
+          .then((data) => {
+            setCity(data[0].name)
+          });
+      });
+    } else {
+      alert("Can't access location");
```
Comment (ttn-ttn) [inline src/App.js:25]: It would be better to set a default location such as New York instead, I think.

File: yarn.lock
Comment (ttn-ttn) [inline yarn.lock:1]: I think you shouldn't commit modifications to yarn.lock

**Comment by akshgpt7** [comment]
@akanshaaa19 can you fix the merge conflicts here?

**Pull request closed without merging**